### ISSUE TITLE
Gradient Descent

### ISSUE THREAD

**Issue by chipbk10** [chipbk10/AI#5]
## Source

- [What is Gradient Descent?](https://www.linkedin.com/posts/shubhangi-sakarkar-66b049112_interview-question-what-is-gradient-descent-activity-7043436894596075520-rhgS/)
- [How Does the Gradient Descent Algorithm Work in Machine Learning?](https://shorturl.at/nSVWY)
- [Gradient Descent Algorithm — a deep dive
](https://towardsdatascience.com/gradient-descent-algorithm-a-deep-dive-cf04e8115f21)
- [Gradient Descent & Backpropagation - Java implementation](https://www.infoworld.com/article/3685492/gradient-descent-in-java.html)
- [Gradient Descent & Ascent in Java](https://azmechatech.medium.com/gradient-descent-ascent-in-java-aaa299cddb73)

**Comment by chipbk10** [comment]
## Leetcode

- [1515. Best Position for a Service Centre](https://leetcode.com/problems/best-position-for-a-service-centre/)
- [must see: gradient descent multiple variable solution](https://leetcode.com/problems/best-position-for-a-service-centre/solutions/842804/python-gradient-descent-with-momentum)
- [69. Sqrt(x)](https://leetcode.com/problems/sqrtx/)
- [gradient descent java implementation](https://www.baeldung.com/java-gradient-descent)
- [Gradient Descent Example for Linear Regression](https://spin.atomicobject.com/gradient-descent-linear-regression/)

**Comment by chipbk10** [comment]
- [derivative](https://www.mathsisfun.com/calculus/derivatives-introduction.html)
- [partial derivative](https://www.mathsisfun.com/calculus/derivatives-partial.html)
